src/dev-utils/view.js

```diff
@@ -914,12 +951,12 @@ function _convertElementNameAndPriority( viewElement ) {
 	throw new Error( `Parse error - cannot parse element's name: ${ viewElement.name }.` );
 }
 
-// Checks if element's type is allowed. Returns `attribute`, `container` or `null`.
+// Checks if element's type is allowed. Returns `attribute`, `container`, `empty`, `widget` or `null`.
 //
 // @param {String} type
 // @returns {String|null}
 function _convertType( type ) {
-	if ( type == 'container' || type == 'attribute' ) {
+	if ( type == 'container' || type == 'attribute' || type == 'empty' || type == 'widget' ) {
```

Comment:
There could be an array with namespaces names and constructors reused in all these conditions instead of checking each type separately.